.github/workflows/ci-app_cloud_e2e_test.yml

```diff
@@ -0,0 +1,174 @@
+name: cloud-testing
+
+# Used to run the e2e tests on lightning.ai
+
+on: # Trigger the workflow on push or pull request, but only for the master branch
+  push:
+    branches:
+      - "master"
+  pull_request:
+    paths:
+      - "src/lightning_app/**"
```

Comment:
Let s remove this and add the trick I added for skipping PL, so we can add some tests as required.

Reply:
well, that conditional slip would need some extra work as seem there is a bug... :)

Reply:
I used this `paths_changes_filter` 

@Borda @tchaton @carmocca could you have a look.

Reply:
Yes, this is a good one. But we might want to do more custom things in the future.